Resources/public/js/components/products/components/attributes/main.js

```diff
@@ -184,12 +184,21 @@ define([
             }.bind(this));
 
             ajaxRequest.complete(function() {
+
+                // set pre selected element in checkbox
+                if (typeof this.attributeTypes[0].name === "string" && typeof this.attributeTypes[0] === "object") {
```

Comment:
use this:

```if (typeof(this.attributeTypes[0]) === "object" && typeof(this.attributeTypes[0].name) === "string") { ```

1. if you call typeof it is more secure in order to avoid syntax errors.
2. By checking if it is an object first, there is no way of having an error like "can not check this.attributeTypes[0].name on undefined"